### ISSUE TITLE
Godot freezes when importing a lot of files from AssetLib

### ISSUE THREAD

**Issue by MeBadDev** [godotengine/godot#68151]
### Godot version

3.5

### System information

Windows 10, GLES3

### Issue description

when godot tries to import alot of assets, it freeze and no response. 
![image](https://user-images.githubusercontent.com/94725241/199411657-8549504a-56a3-4496-a1c5-b656e17e63ce.png)
in this screenshot i am trying to install qodot, but godot crashs.

### Steps to reproduce

1.open asset lib
2.install qodot

**Comment by MeBadDev** [comment]
also is it safe to kill godot while it is importing stuff? I've place it for like 20 minutes and it still don't work

**Comment by Calinou** [comment]
> also is it safe to kill godot while it is importing stuff? I've place it for like 20 minutes and it still don't work

Yes, but files in the `.import/` folder may be corrupted (so you may have to remove that folder and try opening the project again).

**Comment by MeBadDev** [comment]
> How long did you wait for?

I've wait for 15~20 minutes


>What CPU model do you have?

  `Intel(R) Pentium(R) CPU 4415Y @ 1.60GHz   1.61 GHz`

**Comment by terminal-cs** [comment]
> > How long did you wait for?
> 
> I've wait for 15~20 minutes
> 
> > What CPU model do you have?
> 
> `Intel(R) Pentium(R) CPU 4415Y @ 1.60GHz 1.61 GHz`

I know this issue is old but that's your issue, your CPU is very slow, and if you have an HDD then it will be even slower. Though it doesn't help that godot seems to be using a single thread for everything.

**Comment by Calinou** [comment]
> Though it doesn't help that godot seems to be using a single thread for everything.

Resource importing is multithreaded in 4.x, but this can't be backported to 3.x.

**Comment by terminal-cs** [comment]
> > Though it doesn't help that godot seems to be using a single thread for everything.
> 
> Resource importing is multithreaded in 4.x, but this can't be backported to 3.x.

Maybe not? i have an issue with that in 4.1.1: https://github.com/godotengine/godot/issues/79940
